widgetssdk/src/main/java/com/glia/widgets/callvisualizer/controller/CallVisualizerController.kt

```diff
@@ -68,11 +68,12 @@ internal class CallVisualizerController(
         registerCallVisualizerListeners()
     }
 
+    @SuppressLint("CheckResult")
     private fun registerCallVisualizerListeners() {
-        engagementRequestUseCase().unSafeSubscribe { onEngagementRequested() }
-        onIncomingEngagementRequestTimeoutUseCase().unSafeSubscribe { onIncomingEngagementRequestTimeout() }
+        engagementRequestUseCase().subscribe { onEngagementRequested() }
+        onIncomingEngagementRequestTimeoutUseCase().subscribe { onIncomingEngagementRequestTimeout() }
         dialogController.addCallback(::handleDialogState)
-        engagementStartFlow.unSafeSubscribe { dismissVisitorCodeDialog() }
+        engagementStartFlow.subscribe { dismissVisitorCodeDialog() }
```

Comment:
I expected to see adding to disposable. Seems you just switched from `unSafeSubscribe()` to `subscribe()`, isn't it?